subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -358,6 +358,11 @@ Generate a build scan with detailed performance diagnostics.
 
 image::gradle-core-test-build-scan-performance.png[Build Scan performance report]
 
+`--watch-fs`, `--no-watch-fs`::
+Toggles <<gradle_daemon.adoc#sec:daemon_watch_fs,watching the file system>>.
+Allows Gradle to re-use data about the file system in the next build.
```

Comment:
Maybe "metadata" would be a better term here.

Reply:
I am using `information` now.